### ISSUE TITLE
For users who do not have a device Passcode set up, default the Auto Lock setting to "Never"

### ISSUE THREAD

**Issue by nickbrandt** [mozilla-lockbox/lockbox-ios#786]
When a user does not have a device Passcode set up, default the Auto Lock setting to "Never". Tapping into this setting should display a dialog to the user informing them to set up a device Passcode in order to change this setting.

### Visual Design
**InVision**
https://mozilla.invisionapp.com/share/BQEUSO0S2#/328362695_Settings_-_Auto_Lock_-_No_Passcode

**Zeplin**
https://zpl.io/2plM98j

**Comment by joeyg** [comment]
@nickbrandt Currently if the user does not have a device with a Passcode we don't show the AutoLock row at all. Is this task to add the row and show the dialog if tapped?

**Comment by nickbrandt** [comment]
@joeyg yes you are correct. We are bringing this back in to give the user visibility into the autolock functionality, and by tapping on this row (if a user does not have a passcode setup), they would be prompted the same as tapping on the "Lock Now" button with the dialog informing them to set up a passcode. Sorry for not thoroughly explaining that in the description.
